test/per_file_coverage.sh

```diff
@@ -9,8 +9,8 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/common/posix:94.1"
 "source/common/crypto:0.0"
 "source/common/event:93.4" # Emulated edge events guards don't report LCOV
-"source/common/filter:96.5"
-"source/common/filter/http:96.5"
+"source/common/filter:96.3"
+"source/common/filter/http:96.3"
```

Comment:
Is this change related to this PR?

Reply:
The coverage used to drop a bit at a former commit. but the latest commit has 96.5% coverage rate. I reverted this back.